content/100-getting-started/02-setup-prisma/200-add-to-existing-project/110-relational-databases/100-connect-your-database.mdx

```diff
@@ -105,6 +105,29 @@ DATABASE_URL="mysql://root:randompassword@localhost:3306/mydb"
 
 </SwitchTech>
 
+<SwitchTech technologies={['*', 'sqlserver']}>
+
+```prisma file=prisma/schema.prisma
+datasource db {
+  provider = "sqlserver"
+  url      = env("DATABASE_URL")
+}
+```
+
+In this case, the `url` is [set via an environment variable](../../../../concepts/components/prisma-schema#accessing-environment-variables-from-the-schema) which is defined in `.env`:
```

Comment:
Has double colon at end of senteence, but then another sentence before actualy example